model/database/metrics.yaml

```diff
@@ -133,3 +134,39 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "Request Charge consumed by the operation"
+    instrument: histogram
+    unit: "{request_units}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.max_item_count
+    type: metric
+    metric_name: db.cosmosdb.operation.max_item_count
+    brief: "Max item count requested by the feed operation (ex. Query, ReadAll, ReadMany, Change Feed) "
+    instrument: gauge
+    unit: "{count}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.actual_item_count
+    type: metric
+    metric_name: db.cosmosdb.operation.actual_item_count
```

Comment:
I don't see why this metric is cosmos-specific. It could be `db.query.response.item_count` or simiar.

Reply:
renamed as you suggested